website/docs/r/app_configuration.html.markdown

```diff
@@ -11,6 +11,12 @@ description: |-
 
 Manages an Azure App Configuration.
 
+## Disclaimers
+
+-> **Note:** Version 3.27.0 and later of the Azure Provider include a Feature Toggle which will purge an API Management resource on destroy, rather than the default soft-delete. And the Provider will automatically recover a soft-deleted App Configuration during Creation if one is found. See [the Features block documentation](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs#features) for more information on Feature Toggles within Terraform.
```

Comment:
```suggestion
-> **Note:** Version 3.27.0 and later of the Azure Provider include a Feature Toggle which will purge an API Management resource on destroy, rather than the default soft-delete. The Provider will automatically recover a soft-deleted App Configuration during creation if one is found. See [the Features block documentation](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs#features) for more information on Feature Toggles within Terraform.
```